TypeCobol/Tools/APIHelpers/Helpers.cs

```diff
@@ -34,6 +34,7 @@ public static SymbolTable LoadIntrinsic(List<string> paths, DocumentFormat intri
                     parser.Init(path, new TypeCobolOptions { ExecToStep = ExecutionStep.CrossCheck }, intrinsicDocumentFormat);
                     parser.Parse(path);
 
+                    diagnostics.Clear();
                     diagnostics.AddRange(parser.Results.AllDiagnostics());
```

Comment:
As Diagnostics are now cleared at the beggining of each loop, wouldn't it be more clear to write:
```C#
                    var diagnostics = parser.Results.AllDiagnostics();

                    if (diagEvent != null && diagnostics.Count > 0)
                    {
                        foreach(var d in diagnostics)
                        {
                            diagEvent(null, new DiagnosticsErrorEvent() { Path = path, Diagnostic = d });
                        }
                    }
```
The foreach loop is now longer but it makes it clear than diagnostics variable is not reused between iteration of the loop.

Reply:
Done. See 580afedd7028627e429b187961c8dcad51d1d155.